### PR TITLE
Change status group loading and empty state styling

### PR DESCRIPTION
**Pull request by Shadae**
### :nut_and_bolt: Description
- Replace current headings and subheadings with suggested(pro) copywriting from @kagarmoe.
- Changes to the empty state UI design: remove the call-to-action button since this will be a private beta and we will not have published docs by then.

### :+1: Definition of Done
- Global copywriting updates
- Page subheading in all Service Groups pages:

>Service groups are Habitat services ordered by package identifiers and configurations.

- Side panel subheading in all Service Groups pages:

>All services running in a service group.

Screen Shot 2019-06-12 at 8 04 43 PM
Screen Shot 2019-06-12 at 8 06 44 PM

### :athletic_shoe: Demo Script / Repro Steps
Build and start the automate-ui and navigate to the Applications Tab.

After that, you can populate the database and refresh.

### :chains: Related Resources
https://chefio.atlassian.net/browse/A2-772
Closes https://github.com/chef/automate/issues/329

### :white_check_mark: Checklist

- [ ] Necessary tests added/updated?
- [ ] Necessary docs added/updated?
- [ ] Code actually executed?
- [ ] Vetting performed (unit tests, lint, etc.)?

### CODE REVIEW

**Comment by afiune** [comment]
I am taking over this work!

tenor-123942113

**Comment by afiune** [comment]
@apriofrost I agree with @danielsdeleo - We should have a way to display errors in the UI. Do we have any mockups for those cases?

**Comment by afiune** [comment]
I can, at least, print this message: 

image

**Comment by afiune** [comment]
tenor-135677985

**Comment by afiune** [comment]
tenor-135677985

**Comment by afiune** [comment]
@apriofrost and I talked about the error message and we have agreed that we need to do it on a different pull request since it will need to introduce an alert component to notify the user about any error.

File: components/automate-ui/src/app/pages/service-groups/service-groups.component.html [hunk 1]
```diff
@@ -130,12 +128,27 @@
...
+  <!-- Loading State: Displayed only when the main status is loading -->
+  <div class="empty-status" *ngIf="(serviceGroupStatus$ | async) === 'loading'">
+      <chef-loading-spinner size='50'></chef-loading-spinner>
+    <h3>Loading Service Groups...</h3>
+  </div>
+
+  <!-- LoadingFailure State: Displayed only when the main status is a failure -->
+  <div class="empty-status" *ngIf="(serviceGroupStatus$ | async) === 'loadingFailure'">
+      <chef-loading-spinner size='50'></chef-loading-spinner>
+    <h3>Loading Service Groups...</h3>
```
[the remark this answers is not in the document]
Comment (afiune) [reply components/automate-ui/src/app/pages/service-groups/service-groups.component.html]: We will leave it as is, for now, I will create a card to address this error message with an alert component.

File: components/automate-ui/src/app/page-components/services-sidebar/services-sidebar.component.html [hunk 2]
```diff
@@ -61,8 +61,9 @@
         </div>
       </li>
     </ul>
+    <!-- Loading Success + Empty Services: Displayed when there are NO services from the selected filter -->
     <div class="no-services" *ngIf="(services$ | async)?.length === 0">
-      No available services with <b>{{ selectedHealth }}</b> status
+      No services with <b>{{ selectedHealth }}</b> status
```
Comment (apriofrost) [inline components/automate-ui/src/app/page-components/services-sidebar/services-sidebar.component.html]: Could we make the message consistent with the table for empty service groups? 
None of the services returned {{ selectedHealth }}

File: components/automate-ui/src/app/pages/service-groups/service-groups.component.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (danielsdeleo) [reply components/automate-ui/src/app/pages/service-groups/service-groups.component.html]: works for me 👍

**Comment by afiune** [comment]
@apriofrost Updated the messages with your feedback! 💯

**Pull request merged**